core/src/main/resources/db/cgds.sql

```diff
@@ -185,14 +183,13 @@ CREATE TABLE `patient_list` (
 -- --------------------------------------------------------
 
 --
--- Table structure for table `patient_list_list`
+-- Table structure for table `sample_list_list`
 --
--- TODO rename the PATIENT_ID column, which is misleading
-CREATE TABLE `patient_list_list` (
+CREATE TABLE `sample_list_list` (
   `LIST_ID` int(11) NOT NULL,
-  `PATIENT_ID` int(11) NOT NULL,
-  PRIMARY KEY  (`LIST_ID`,`PATIENT_ID`),
-  FOREIGN KEY (`PATIENT_ID`) REFERENCES `sample` (`INTERNAL_ID`) ON DELETE CASCADE
+  `SAMPLE_ID` int(11) NOT NULL,
+  PRIMARY KEY  (`LIST_ID`,`SAMPLE_ID`),
+  FOREIGN KEY (`SAMPLE_ID`) REFERENCES `sample` (`INTERNAL_ID`) ON DELETE CASCADE
```

Comment:
@jjgao : the FOREIGN KEY will ensure the integrity. ON DELETE CASCADE will just *automate* the deletion cascade, so when you want to delete a sample, all you have to do is `delete from sample where...` . This is a great feature if the cascade does not involve deleting huge amount of records. I am just worried that this will also be used to delete a complete study...I am not sure if DB can handle this well....